mandatory/3-function-output.js

```diff
@@ -1,17 +1,19 @@
-// Add comments to explain what this function does. You're meant to use Google!
+//My Comment - This function gives a random number by using math.random() and multiplying the value by 10  
```

Comment:
Can you explain the range of the random numbers that will be returned? What range does Math.random() return, and so what will your method return?